addon/mixins/osf-token-login-route.js

```diff
@@ -33,6 +35,6 @@ export default Ember.Mixin.create(UnauthenticatedRouteMixin, {
         }
 
         return this.get('session').authenticate('authenticator:osf-token', accessToken)
-            .then(() => this.transitionTo('index'));
+            .catch(err => console.log('Authentication failed: ', err));
```

Comment:
Mostly by design. Since the actual login form happens by going to an external URL, this mixin becomes much less responsible for logging someone in. Basically, the beforeModel hook is used to set up a page if someone is logged in.

If they're not logged in, that's ok too- it just means they see something a little different on the page. The underlying design assumption is that CAS controls the login page, and the ember mixin exists just to grab an access token and marking the page as logged in where appropriate.

Currently, CAS limits the redirectURI to a single specific URL, so this mixin could reasonably be used in, eg, just the index route of the site.